src/Chapter1.hs

```diff
@@ -653,7 +670,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n > 10 = firstDigit (div n 10)
+  | otherwise = n
```

Comment:
Unfortunately, this will return negative numbers on negative input...
Also, I think you could include `10` in the previous line comparison 👍🏼

Reply:
thanks for the reply, on second attempt I've implemented this function using let a = abs n
```
firstDigit :: Int -> Int
firstDigit n =
  let a = abs n
      b = if a >= 10 then firstDigit (div a 10) else a
  in b
```